.github/workflows/fuzz.yml

```diff
@@ -53,6 +53,7 @@ hashes_sha1,
           override: true
           profile: minimal
       - name: fuzz
+        run: if [[ "${{ matrix.fuzz_target }}" =~ ^bitcoin ]]; then export RUSTFLAGS='--cfg=hashes_fuzz --cfg=secp256k1_fuzz'; fi
```

Comment:
Are you sure the env var is kept across multiple `run:` statements? We should debug print it I guess.